src/main/java/edu/harvard/iq/dataverse/api/Admin.java

```diff
@@ -2541,4 +2521,162 @@ public Response getFeatureFlag(@PathParam("flag") String flagIn) {
         }
     }
 
+	@GET
+	@AuthRequired
+	@Path("/datafiles/auditFiles")
+	public Response getAuditFiles(@Context ContainerRequestContext crc,
+								  @QueryParam("firstId") Long firstId, @QueryParam("lastId") Long lastId,
+								  @QueryParam("datasetIdentifierList") String datasetIdentifierList) throws WrappedResponse {
```

Comment:
@reviewdog seems to be sleeping on the job but this new method is full of tabs. Please reformat with Netbeans or otherwise switch to spaces. Please see https://guides.dataverse.org/en/6.4/developers/coding-style.html#tabs-vs-spaces

Reply:
https://github.com/IQSS/dataverse/actions/runs/11922455279/job/33228743537?pr=11016 says, Checkstyle ends with 254 errors." I'm not sure why they aren't appearing here. @reviewdog found tabs in late July: https://github.com/IQSS/dataverse/pull/10729#discussion_r1693220572

@poikilotherm you set up this action back in #7911. Any insight?

Reply:
Wow, you reformatted the entire file. At https://guides.dataverse.org/en/6.4/developers/coding-style.html#format-code-you-changed-with-netbeans we ask for reformatting of only changed code.

@stevenwinship you've worked on large teams. I'm curious what you think of this "gofmt for Java" post of mine: https://groups.google.com/g/dataverse-dev/c/y2Jpk3szTf8/m/rckKmP6-BgAJ

Should we talk about it at tech hours?

In short, what I want is a tool we all use rather to keep code consistent, rather than what we have now, which is an inconsistent mess. Over at https://github.com/gdcc/xoai we use Spotless.

Anyway, I'll move this PR along but I'm happy to hear your thoughts! There's the dev list thread above, a new-ish thread at https://dataverse.zulipchat.com/#narrow/channel/379673-dev/topic/code.20formatting.20.28Spotless.2C.20Checkstyle.2C.20etc.2E.29/near/432974039 or whatever.

Reply:
Bill McKinney's reply to you is what we used at Nokia. A style xml file that could be plugged into Eclipse and IntelJ. With over 60 engineers touching the code it was nice to have a fixed format.

FWIW I did a global find/replace of tabs to spaces. I'm sorry it reformatted the entire file but at least it's consistent now.

Reply:
Oh. and we did block code from being checked in with tabs. But we were using garret and not GitHub or GitLab at the time.